tests/core/test_plugin_download_security.py

```
@@ -0,0 +1,82 @@
+
+        pm = PluginManager()
+        dest_path = Path("test_timeout.zip")
+
+        # We assume the call fails or succeeds, but we check if timeout was passed
+        mock_response = AsyncMock()
+        mock_response.aiter_bytes = AsyncMock(return_value=[])
+        mock_client.stream.return_value.__aenter__.return_value = mock_response
+
+        try:
+            await pm._download_file("https://example.com/slow.zip", dest_path)
+        except:
```

Ruff (0.15.1)

[error] Do not use bare `except`

(E722)